$NetBSD: patch-aa,v 1.10 2010/11/05 22:20:57 tonio Exp $

AI_ADDRCONFIG doesn't exist on some systems, such as netbsd 5.0.2
Pathc from revision 714 of bitlbee

--- lib/proxy.c.orig	2010-10-22 00:51:31.000000000 +0000
+++ lib/proxy.c
@@ -52,6 +52,9 @@ char proxypass[128] = "";
 #ifndef AI_NUMERICSERV
 #define AI_NUMERICSERV 0
 #endif
+#ifndef AI_ADDRCONFIG
+#define AI_ADDRCONFIG 0
+#endif
 
 struct PHB {
 	b_event_handler func, proxy_func;
